nano/lib/stats.cpp

```diff
@@ -10,36 +10,6 @@
 #include <fstream>
 #include <sstream>
 
-nano::error nano::stat_config::deserialize_json (nano::jsonconfig & json)
```

Comment:
The header file declaration needs removing too.
`nano::error deserialize_json (nano::jsonconfig & json);`